docs/content/en/docs/implementing/metrics.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Keptn Metrics
+description: Implement Keptn site metrics
+weight: 130
+---
+
+Kubernetes provides two metrics servers,
+one that is custom, the other external.
+However, the Kubernetes metrics servers
+only allow you to release a single service on a API;
+you cannot use multiple observability platforms
+in one namespace for one API.
```

Comment:
Oh dear!  I think these are Giovanni's words but perhaps I messed them up.  The intent is to explain why KLT has its own metrics implementation rather than using the Kubernetes metrics

Reply:
So are you saying no mention of the k8s metrics servers?  I left in the initial sentence for now -- it seems that we need to at least mention the k8s metrics servers or people are going to wonder...  But maybe different content.

What if we lead off with the "The Keptn metric server..." sentence and then say that it is an overlay to the k8s metric servers?  Is that accurate?